mlrun/serving/routers.py

```diff
@@ -41,14 +40,7 @@
 from .utils import RouterToDict, _extract_input_data, _update_result_body
 from .v2_serving import _ModelLogPusher
 
-
-class ExecutorTypes:
-    thread = "thread"
-    process = "process"
-
-    @staticmethod
-    def all():
-        return [ExecutorTypes.thread, ExecutorTypes.process]
+local_routes = {}
```

Comment:
please add a comment for the reason you use it as a global variable and its use

Reply:
doesn't exist anymore